File: set files/set.py
import time


f = open("vector104.txt","r")
lines = [line.rstrip('\n') for line in f]

a = []
for i in lines:
    a.append(int(i))
start = time.time()
a =set(a)
stop = time.time() - start
print(stop)
f.close()

File: set files/set.php
<?php
require 'vendor/autoload.php';

header('Content-type: text/plain');
$myfile = fopen("vector104.txt", "r") or die("Unable to open file!");
$n = 10*1000;
$start = microtime(true);
$hash_set = new \Ds\Set();
while (!feof($myfile)) {
    $line = fgets($myfile);
    $hash_set -> add($line);
}
fclose($myfile);
$stop = microtime(true)-$start;
echo $stop;
?>